s3/sc-s3-synapse-ra.yaml

```diff
@@ -25,8 +25,9 @@ Parameters:
     Description: (Optional) Name of the created bucket.
     Default: ""
 Conditions:
-  HasBucketName: !Not [!Equals [!Ref BucketName, ""]]
+  HasSynapseUserName: !Not [!Equals [!Ref SynapseUserName, ""]]
```

Comment:
This is a good thought but I don't think it's necessary.  The purpose of this template is to create a synapse bucket product so a SynapseUserName is required otherwise it won't be a synapse bucket.  

We should also remove the `default: ""` from properties section.

Reply:
fair enough... wanting to avoid future bugs